src/shared/services/map-categories/index.js

```diff
@@ -0,0 +1,28 @@
+function mapCategories(data) {
+  const categories = [];
+  const subcategories = [];
+
+  if (data && data.results) {
+    data.results.map((category) => {
+      categories.push({
+        key: category._links && category._links.self && category._links.self.href,
+        value: category._display
+      });
+
+      category.sub_categories.map((subcategory) => {
+        subcategories.push({
+          key: subcategory._links && subcategory._links.self && subcategory._links.self.href,
+          value: subcategory._display
+        });
+
+        return true;
```

Comment:
Return true is not necessary in a map, right?

Reply:
nee klopt: weggehaald

Reply:
nee de Linter valt erover als er geen returns zijn:
   6:33  error  Expected to return a value in arrow function  array-callback-return
wat zal ik doen?

Reply:
You could use .forEach
(https://eslint.org/docs/rules/array-callback-return)

Reply:
gedaan